api/app/GraphQL/Queries/CountPoolCandidatesByPool.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace App\GraphQL\Queries;
+
+use App\Models\PoolCandidate;
+use App\Models\User;
+use Database\Helpers\ApiEnums;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Support\Facades\Log;
```

Comment:
Symbol 'App\GraphQL\Queries\Log' is declared but not used.

Reply:
Thanks!  Fixed in [Tidy up based on reviews](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4455/commits/74587e85db9aeb7ae9cd36053f21c2d005a4081a).